web/src/lib/components/utilities-page/duplicates/duplicate-asset.svelte

```diff
@@ -51,6 +52,7 @@
     fileName: isDifferent((a) => a.originalFileName),
     fileSize: isDifferent((a) => getFileSize(a)),
     resolution: isDifferent((a) => getAssetResolution(a)),
+    originalPath: isDifferent((a) => a.originalPath ?? 'unknown'),
```

Comment:
I think you want to fall back to `$t('unknown')` here, right?

```suggestion
    originalPath: isDifferent((a) => a.originalPath ?? $t('unknown')),
```